static/src/js/components/AccessMethod/SwodlrForm.jsx

```diff
@@ -0,0 +1,439 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Accordion,
+  Card,
+  Form,
+  Container,
+  Row,
+  Table,
+  OverlayTrigger,
+  Tooltip,
+  Col
+} from 'react-bootstrap'
+
+import { FaQuestionCircle, FaChevronDown } from 'react-icons/fa'
+
+import {
+  swodlrToolTips,
+  utmRasterOptions,
+  geoRasterOptions
+} from '../../constants/swodlrConstants'
+
+import EDSCIcon from '../EDSCIcon/EDSCIcon'
+import ProjectPanelSection from '../ProjectPanels/ProjectPanelSection'
+
+import './SwodlrForm.scss'
+
+/**
+   * Renders Swodlr Form.
+   * @param {Object} props - The props passed into the component.
+   * @param {Array} props.granuleList - The list of selected granules for the current collection.
+   * @param {String} props.collectionId - The project collection id.
+   * @param {Function} props.onUpdateAccessMethod - Function to update metadata for the currently selected access method.
+   * @param {String} props.selectedAccessMethod - The currently seletected access method.
+   * @param {Function} props.setGranuleList - Function to update the granule list of the parent component.
+  */
+
+const SwodlrForm = ({
+  granuleList,
+  collectionId,
+  onUpdateAccessMethod,
+  selectedAccessMethod,
+  setGranuleList
+}) => {
+  const [granuleExtent, setGranuleExtent] = useState(false)
+  const [sampleGrid, setSampleGrid] = useState('UTM')
+  const [rasterResolution, setRasterResolution] = useState(90)
+
+  const handleSwoldrOptions = () => {
```

Comment:
Each one of these methods should have a comment explaining what its usage is. We don't typically use full jsdoc style blocks, but at least one line explaining something helps understand it when we look at it later.

handleSwoldrOptions, handleCollectionGranuleListUpdate, handleRasterResolutionUpdate, handleSampleGrid, handleGranuleExtent and the useEffect